### PR TITLE
feat: Mesh in Konnect changelog

### PR DESCRIPTION
**Pull request by cloudjumpercat**
### Description

 MinK was missing a changelog, so I added one. 

DOCU-3479
### Testing instructions

Preview link: 

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

**Review by lena-larionova — CHANGES_REQUESTED** [review]
You should use the [release note in Aha](https://konghq.aha.io/epics/KP-E-387) as the basis for the changelog entry (for example, you can take out things like the first sentence, which is clearly a marketing spin):

> Kong Konnect takes a major step forward, becoming the industry’s first unified API lifecycle management platform. Kong Mesh joins Kong Gateway and Kong Ingress Controller as the generally available runtime engines that are managed centrally by Kong Konnect’s management plane. Users can now identify mesh services, view relevant metadata (mesh, zone, etc.), and access the service's details in Mesh Manager through Konnect. This is the industry’s first unified API platform where all configuration management is in a single location with a clean and modern interface to manage services from Kong Gateway, Kong Ingress Controller, and Kong Mesh. 
>
> Konnect users can now navigate to a new Mesh Manager menu to create, modify, and delete mesh global control planes (MGCP). MGCP, as the name suggests, is a Control Plane mapped to different environments or business units. Each MGCP maps to a Kubernetes or Linux zone through a process that is similar to adding a Control Plane for gateway. The initial onboarding, management, and view will look similar in Mesh Manager to that of Gateway Manager.
>
> Organizations can lower their operational overhead and elevate their developers’ experience by streamlining the operation of Kong Gateways, Kong Ingress Controller and by simplifying the deployment of Kong Mesh with a hosted control plane that can be implemented across any cloud, virtual machines, or on-premises solution.

File: app/konnect/updates.md [hunk 1]
```diff
@@ -37,6 +37,11 @@ services. [Try it today!](https://cloud.konghq.com/quick-start)
 
 : The Gateway Control Plane Overview page is now updated to present the users all relevant information of a specific control plane. In addition, prompts that nudge users to create their first service, route, plugin, and consumer are presented on the Overview Pages to nudge users to immediately dive into Konnect for the first time.
 
+**{{site.mesh_product_name}} in {{site.konnect_short_name}}**
```
Comment (cloudjumpercat) [inline app/konnect/updates.md:40]:
```suggestion
**{{site.mesh_product_name}} in {{site.konnect_short_name}} (Enterprise and new Plus customers)**
```

File: app/konnect/updates.md [hunk 2]
```diff
@@ -37,6 +37,11 @@ services. [Try it today!](https://cloud.konghq.com/quick-start)
 
 : The Gateway Control Plane Overview page is now updated to present the users all relevant information of a specific control plane. In addition, prompts that nudge users to create their first service, route, plugin, and consumer are presented on the Overview Pages to nudge users to immediately dive into Konnect for the first time.
 
+**{{site.mesh_product_name}} in {{site.konnect_short_name}} (Enterprise and new Plus customers)**
```
Comment (Guaris) [inline app/konnect/updates.md:40]:
```suggestion
**{{site.mesh_product_name}} in {{site.konnect_short_name}}**
```

Comment (Guaris) [reply app/konnect/updates.md:40]: I think this needs to be explicit and point to the pricing page. 
As in:
* Enterprise will be able to access mesh immediately and should contact their sales representative. 
* New customers can apply their trial credits towards mesh zones 
For more information view [pricing](pricing). 
Mesh is not available for existing plus customers, stay tuned for more information. 

I would clear this messaging with someone else but explicit is better than not.

**Review by Guaris — CHANGES_REQUESTED** [review]

File: app/konnect/updates.md [hunk 3]
```diff
@@ -37,6 +37,17 @@ services. [Try it today!](https://cloud.konghq.com/quick-start)
...
+
+: You can use [Mesh Manager](/konnect/mesh-manager/) to create, modify, and delete mesh global control planes. Mesh global control planes are control planes that are mapped to different environments or business units. Each mesh global control plane maps to a Kubernetes or Linux zone through a process that is similar to adding a control plane for {{site.base_gateway}} using Gateway Manager. You can also now identify [service meshes](/mesh/latest/introduction/about-service-meshes/), view relevant metadata like meshes and zones, and access a service's details in {{site.konnect_short_name}}.
+
+: Mesh Manager is available now to Enterprise and new {{site.konnect_short_name}} customers:
+  * Enterprise customers will be able to access Mesh Manager immediately and can contact their sales representative for more information.
+  * New customers can apply their trial credits towards mesh zones.
+    For more information, see [{{site.konnect_short_name}} pricing](https://konghq.com/pricing).
+    Mesh Manager is not available for existing plus customers. Stay tuned for more information.
```
Comment (cloudjumpercat) [inline app/konnect/updates.md:49]: @johnharris85 Could you review the wording of this to make sure this is accurate/everything we want to say for the changelog?

**Review by lena-larionova — APPROVED** [review]

**Review by Guaris — APPROVED** [review]

**Pull request merged**